File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/rgisbert.py
def fizzBuzz(n):
    isMultipleOf3 = n % 3 == 0
    isMultipleOf5 = n % 5 == 0

    if (isMultipleOf3 and isMultipleOf5):
        return 'fizzbuzz'
    if (isMultipleOf3):
        return 'fizz'
    if (isMultipleOf5):
        return 'buzz'
    return n


for num in range(1, 101):
    print(fizzBuzz(num))

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/javascript/rgisbert.js
const fizzBuzz = ( num ) => {
    const isMultipleOf3 = num % 3 === 0;
    const isMultipleOf5 = num % 5 === 0;

    if( isMultipleOf3 && isMultipleOf5 ) return 'fizzbuzz';
    if( isMultipleOf3 ) return 'fizz';
    if( isMultipleOf5 ) return 'buzz';
    return num;
}

for(let num = 1; num <= 100; num++) {
    console.log( fizzBuzz( num ) );
}